package.json

```diff
@@ -53,6 +54,8 @@
     "koa2-cors": "^2.0.6",
     "lodash": "^4.17.11",
     "log4js": "^4.2.0",
+    "newrelic": "^5.8.0",
```

Comment:
rm

Reply:
No need, I'll take care of it when rebasing

Reply:
❓

Reply:
What do you mean @alkiskal . It shouldn't be added again.